icu4c/source/i18n/smpdtfmt.cpp

```diff
@@ -2561,10 +2561,10 @@ SimpleDateFormat::parse(const UnicodeString& text, Calendar& cal, ParsePosition&
             if (btz != NULL) {
                 if (tzTimeType == UTZFMT_TIME_TYPE_STANDARD) {
                     btz->getOffsetFromLocal(localMillis,
-                        BasicTimeZone::kStandard, BasicTimeZone::kStandard, raw, dst, status);
+                        (UTimeZoneLocalOption)BasicTimeZone::kStandard, (UTimeZoneLocalOption)BasicTimeZone::kStandard, raw, dst, status);
```

Comment:
Do we need to pass in precisely 0x01? Can we pass in one of the two UCAL_TZ_LOCAL_STANDARD_* values? If not, do we need more UTimeZoneLocalOption constants? @yumaoka

Reply:
This call can use UTimeZoneLocalOption enums as below.

```
btz->getOffsetFromLocal(localMillis,
    UCAL_TZ_LOCAL_STANDARD_FORMER, UCAL_TZ_LOCAL_STANDARD_LATTER, raw, dst, status);
```

Original code uses BasicTimeZone::kstandard to pick which side of time is used. In case of both side of transitions are same - standard or daylight, former offset is used for non-existing wall time slot, and latter offset is used for duplicated wall time slot.